aten/src/ATen/native/native_functions.yaml

```diff
@@ -5599,13 +5599,7 @@
     CUDA: leaky_relu
     QuantizedCPU: quantized_leaky_relu
 
-- func: leaky_relu_backward.grad_input(Tensor grad_output, Tensor self, Scalar negative_slope, *, Tensor(a!) grad_input) -> Tensor(a!)
-  python_module: nn
-  dispatch:
-    CPU: leaky_relu_backward_out
-    CUDA: leaky_relu_backward_out
-
-- func: leaky_relu_backward(Tensor grad_output, Tensor self, Scalar negative_slope) -> Tensor
+- func: leaky_relu_backward(Tensor grad_output, Tensor self, Scalar negative_slope, bool self_is_result=False) -> Tensor
```

Comment:
why do we have default arguments for backward?  You call backward functions explicitly anyway, i.e. it's not a user API, so we should just always pass every argument.